services-js/commissions-app/src/client/common/Checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export interface CheckboxProps {
+  name: string;
+  id: any;
+  label: string;
+  value: any;
+  onChange: any;
+  onBlur: any;
+  key: any;
+}
+
+export default class Checkbox extends React.Component<CheckboxProps> {
+  render() {
+    return (
+      <div>
+        <label className="cb">
+          <input
+            id={this.props.id}
+            name="commissions"
+            value="commissions"
```

Comment:
Also, it's a good idea to try and make this component general. It could be used for more that just a commission, like a "remember me" checkbox, so this file shouldn’t say "commission" anywhere in it.